packages/tabs/src/Tabs.tsx

```diff
@@ -162,27 +162,23 @@ function Tabs({
   useEventListener('keydown', handleArrowKeyPress);
 
   function calcStyle() {
-    if (
-      !tabListRef ||
-      !tabListRef.current ||
-      typeof currentIndex !== 'number'
-    ) {
+    if (!tabListRef || typeof currentIndex !== 'number') {
       return null;
     }
 
     const tabListChildren: Array<Element> = Array.from(
-      tabListRef.current.children,
-    );
+      tabListRef.children,
+    ).filter(child => child != null);
 
     let computedX = 0;
 
     for (let i = 0; i < currentIndex; i++) {
-      computedX += tabListChildren[i].scrollWidth;
+      computedX += tabListChildren[i]?.scrollWidth;
```

Comment:
feels sketchy to add `undefined` to a number. could we just wrap this in `if (i in tabListChildren)`?